CHANGELOG.md

```diff
@@ -21,6 +21,7 @@
 * Implement return to Choose Job Profile page functionality (UIDATIMP-103)
 * Write tests for return to Choose Job Profile page functionality (UIDATIMP-154)
 * Implement file extension editing feature (UIDATIMP-84)
+* Fix bugs (UIDATIMP-159 and UIDATIMP-160)
```

Comment:
Please name commits more descriptively.

For example for UIDATIMP-159 the message would be. Apply the same approach for the CHANGELOG.
UIDATIMP-159: Handle new/updated file extensions alphabetical order in the results list issue